src/typeDefs/queries.ts

```diff
@@ -67,7 +67,7 @@ export const queries = gql`
     getlanguage(lang_code: String!): [Translation]
 
     getPlugins: [Plugin]
-    getAdvertisements: [Advertisement]
+    advertisements: [Advertisement]
```

Comment:
you are already exposing advertisements on Organization, is this field necessary? One use case would be accessing all advertisements for all organizations. If that is the case make this field `AdvertisementsConnection` too.